updatehub/tests/failed_integration_test.rs

```diff
@@ -65,55 +65,18 @@ fn failing_invalid_download_dir() {
     "###);
 
     insta::assert_snapshot!(remove_carriage_newline_characters(output_client), @r###"
-    Ok(
-        Response {
-            update_available: true,
-            try_again_in: None,
-        },
-    )
-    "###);
+    Response {
+        update_available: true,
+        try_again_in: None,
+    }"###);
```

Comment:
I believe this might be improved as well. Maybe we could print something like:

```
Update available. The update is running in background.
```

but we also need to handle the other cases, such as when we receive:

```rust
Response {
        update_available: false,
        try_again_in: None,
}
```

and when it receives:

```rust
Response {
        update_available: false,
        try_again_in: Some(...),
}
```

Reply:
Sure, i'll make a another PR to improve the output for probe command.